app/views/filters/_form.html.erb

```diff
@@ -56,7 +63,7 @@
     <% if show_location_tab? %>
       <div class="tab-pane" id="locations">
         <% collection = User.current.locations.empty? ? Location : User.current.locations %>
-        <%= multiple_checkboxes f, :locations, f.object, collection, :label => _("Locations") %>
+        <%= multiple_checkboxes f, :locations, f.object, collection, {}, :label => _("Locations") %>
```

Comment:
It's an issue in develop too indeed, pretty nasty.  http://projects.theforeman.org/issues/16310